### PR TITLE
darkmode?

### PR DESCRIPTION
**Pull request by luckasw**

### CODE REVIEW

**Comment by kevinkliimask** [comment]
https://github.com/pacocoursey/next-themes

**Comment by luckasw** [comment]
> https://github.com/pacocoursey/next-themes

i see...

**Comment by luckasw** [comment]
Hey, in the spirit of Christmas and giving, could you please review and merge this PR? It’d be greatly appreciated!

**Comment by kevinkliimask** [comment]
@luckasw vaata minu commit üle

**Comment by kevinkliimask** [comment]
not found lehega mingi jama

File: components/theme-switcher.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+const ThemeSwitcher = () => {
+  const [theme, setTheme] = useState(localStorage.getItem('theme') || 'light');
+
+  useEffect(() => {
+    document.documentElement.classList.toggle('dark', theme === 'dark');
```
Comment (luckasw) [inline components/theme-switcher.tsx:9]: Proovib siin localstorageit buildi ajal katte saada.

**Pull request merged**